assets/src/components/account/GettingStartedOverview.tsx

```diff
@@ -8,6 +8,7 @@ import * as API from '../../api';
 import {User} from '../../types';
 import {Paragraph, Input, colors, Text, Title} from '../common';
 import {BASE_URL} from '../../config';
+import * as _ from 'lodash';
```

Comment:
let's just `import {debounce} from 'lodash'` if that's the only function we're using :) 

(i think this will also help optimize the webpack build to not import all of lodash)